py.typed marker is missing in 0.34

Hey, since the release of 0.34 mypy is complaining with 

```
Run tox -e type_check
##[debug]/usr/bin/bash -e /home/runner/work/_temp/bd19b5f4-fb3e-460f-b257-cb95c16b7b7d.sh
type_check: install_deps> python -I -m pip install mypy types-python-dateutil types-pytz -r requirements.txt
type_check: freeze> python -m pip freeze --all
type_check: holidays==0.34,mypy==1.5.1,mypy-extensions==1.0.0,pip==23.2.1,python-dateutil==2.8.2,pytz==2023.3.post1,setuptools==68.2.0,six==1.16.0,types-python-dateutil==2.8.19.14,types-pytz==2023.3.1.1,typing_extensions==4.8.0,wheel==0.41.2
type_check: commands[0]> mypy --show-error-codes src/bdew_datetimes
src/bdew_datetimes/calendar.py:7: error: Skipping analyzing "holidays": module is installed, but missing library stubs or py.typed marker  [import]
src/bdew_datetimes/calendar.py:8: error: Skipping analyzing "holidays.constants": module is installed, but missing library stubs or py.typed marker  [import]
src/bdew_datetimes/calendar.py:9: error: Skipping analyzing "holidays.countries.germany": module is installed, but missing library stubs or py.typed marker  [import]
src/bdew_datetimes/calendar.py:9: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
src/bdew_datetimes/periods.py:14: error: Skipping analyzing "holidays": module is installed, but missing library stubs or py.typed marker  [import]
Found 4 errors in 2 files (checked 4 source files)
type_check: exit 1 (1.43 seconds) /home/runner/work/bdew-datetimes/bdew-datetimes> mypy --show-error-codes src/bdew_datetimes pid=1834
  type_check: FAIL code 1 (6.75=setup[5.32]+cmd[1.43] seconds)
  evaluation failed :( (6.82 seconds)
Error: Process completed with exit code 1.
```
The following screenshot shows both tarballs from pypi:

![grafik](https://github.com/vacanza/python-holidays/assets/20128340/7a250af9-a401-4781-ad22-3b850a5b7f69)


Comparing both releases show that with ac29789 `include holidays/py.typed` in MANIFEST.in was dropped.

---

Origin: https://github.com/mj0nez/bdew-datetimes/pull/55

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,5 +1,6 @@
 include CHANGES
 include CONTRIBUTING.rst
+include holidays/py.typed
 include Makefile
 
 recursive-include docs *

